features/steps/room_steps.js

```diff
@@ -15,9 +18,16 @@ When("a {actor} provides the correct {room} Key", function (actor, room) {
   return "pending";
 });
 
-Then("the {actor} is placed in the {room}", function (actor, room) {
-  // Write code here that turns the phrase above into concrete actions
-  return "pending";
+When('the {actor} taps the {room} in the Room Picker', async function (actor, room) {
+  const workspacePage = new WorkspacePage(this.driver)
+  const roomCard = workspacePage.findRoomCard(room);
+  await workspacePage.enterRoom(roomCard);
+});
+
+Then("the {actor} is placed in the {room}", async function (actor, room) {
+  await this.driver.wait(until.elementLocated(By.css("[name='jitsiConferenceFrame0']")));
+  const jitsiIframe = await this.driver.findElements(By.css("[name='jitsiConferenceFrame0']"))
+  assert.equal(jitsiIframe.length, 1, 'Jitsi iframe not found');
```

Comment:
This feels like something belongs in the WorkspacePage object, i.e. `assert.equal(await this.workspace.videoPanel().isPresent())` to keep calls to the driver directly out of step definitions.